fish-mode.el

```diff
@@ -387,7 +389,7 @@ For example, (fold F X '(1 2 3)) computes (F (F (F X 1) 2) 3)."
 
 (defun fish/line-contains-open-switch-term? ()
   "Returns t if line contains switch term, nil otherwise."
-  (> (fish/count-of-tokens-in-string (rx symbol-start "switch" symbol-end) nil (fish/current-line))
+  (> (fish/count-of-tokens-on-current-line (rx symbol-start "switch" symbol-end) nil)
```

Comment:
`fish/count-of-tokens-on-current-line` should be `fish/count-tokens-on-current-line`